tests/unit_test/llm_test.py

```
@@ -258,10 +262,14 @@ async def test_gpt3_faq_embedding_train_payload_with_int(self, mock_embedding, a
         )
         llm_secret.save()
 
-        embedding = list(np.random.random(LLMProcessor.__embedding__))
+        embedding = [[0.1] * 3072]
         input = {"name": "Ram", "color": "red"}
```

Ruff (0.14.14)

[error] Local variable `input` is assigned to but never used

Remove assignment to unused variable `input`

(F841)